app/views/foods/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= form_with(model: food) do |form| %>
+   <% if food.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(food.errors.count, "error") %> prohibited this food from being saved:</h2>
+
+      <ul>
+        <% food.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name , class: 'form-control'%>
+  </div>
+
+  <div>
+    <%= form.label :measurement_unit, style: "display: block" %>
+    <%= form.text_field :measurement_unit, class: 'form-control' %>
+  </div>
+
+  <div>
+    <%= form.label :price, style: "display: block" %>
+    <%= form.number_field :price, class: 'form-control' %>
+  </div>
+
+  <div>
+    <%= form.label :quantity, style: "display: block" %>
+    <%= form.number_field :quantity, class: 'form-control' %>
+  </div>
+
+  <div>
+    <%= form.label :user_id, style: "display: block" %>
+    <%= form.text_field :user_id, class: 'form-control' %>
+  </div>
```

Comment:
[OPTIONAL] It would be best if you did not request for `user_id` when creating a new food. Chances are that the user does not know their `user_id` since that's auto-generated by the database and could be any random number.

- This affects user experience since users cannot create foods if they do not know their id.
- You could use Device's `current_user` in the `create` method in the `foods_controller` to referrence the current user id instead. Something like `@food = current_user.foods.new(food_params)`.